{
    "type": "bundle",
    "id": "bundle--9a6b2acb-9c09-4b2d-8afc-3e3c1729f82d",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--fa48ecc3-4615-4ec5-998d-a155f8d154c0",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-03-26T00:39:39.434123Z",
            "modified": "2025-03-26T00:39:39.434123Z",
            "name": "CVE-2025-2312",
            "description": "A flaw was found in cifs-utils. When trying to obtain Kerberos credentials, the cifs.upcall program from the cifs-utils package makes an upcall to the wrong namespace in containerized environments. This issue may lead to disclosing sensitive data from the host's Kerberos credentials cache.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-2312"
                }
            ]
        }
    ]
}